mono/mini/aot-runtime.c

```diff
@@ -2760,10 +2763,14 @@ compute_llvm_code_range (MonoAotModule *amodule, guint8 **code_start, guint8 **c
 static gboolean
 is_llvm_code (MonoAotModule *amodule, guint8 *code)
 {
+#if HOST_WASM
```

Comment:
In the interest of the "retargetable runtime", can these be `if` instead of `#if`?

Reply:
Those are workarounds for now to unblock further work. I don't know what we'll end up with.
I rather not have any of those at all.